src/default-options-handler.ts

```diff
@@ -46,4 +52,23 @@ function addPullRequestDefaults(
   return pullRequestSettings;
 }
 
-export {addPullRequestDefaults};
+/**
+ * Format user input for pull request review comments
+ * @param options The user's options input for review comments
+ * @returns the formatted version of user input for pull request review comments
+ */
+export function addRevuewCommentsDefaults(
+  options: CreateReviewCommentUserOptions
+) {
+  const reviewCommentSettings: CreateReviewComment = {
+    repo: options.repo,
+    owner: options.owner,
+    pullNumber: options.pullNumber,
+    // if zero set as 0
+    pageSize:
+      options.pageSize === null || options.pageSize === undefined
+        ? DEFAULT_PAGE_SIZE
+        : options.pageSize,
+  };
+  return reviewCommentSettings;
```

Comment:
Let's declare the return type in the signature and then we can return the object directly.